FileSortService/Controllers/FileSortController.cs

```diff
@@ -87,5 +87,51 @@ public JsonResult SaveFile()
                 return new JsonResult("anon.png");
             }
         }
+
+        [Route("DeleteFile")]
```

Comment:
you don't need to use this rote like 'DeleteFile' please delete

Reply:
Done